pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -1000,3 +1073,81 @@ ANONYMIZED
 	}
 	return sb.String()
 }
+
+// MinimalNodeInfo contains the most essential information about a node
+type MinimalNodeInfo struct {
+	ProviderID string `json:"providerID"`
+	Image      string `json:"image"`
+}
+
+// RunningImages assigns information about running containers to a specific image index.
+// The index is a reference to an item in the related `ContainerImageSet` instance.
+type RunningImages map[int]int
+
+// PodsWithAge maps the YYYY-MM string representation of start time to list of pods running since that month.
+type PodsWithAge map[string]RunningImages
+
+// Add inserts the specified container information into the data structure.
+func (p *PodsWithAge) Add(startTime time.Time, image int) {
+	const YyyyMmFormat = "2006-01"
+	month := startTime.UTC().Format(YyyyMmFormat)
+	if imageMap, exists := (*p)[month]; exists {
```

Comment:
It is not a problem, but method with pointer reciver will automatically use pointer to variable instead of a varable. It is go convention.  So in this case, you can use p[month] and later on line 1101 p[mont] = RunningImages{image: i} will work too.

Reply:
When I try to change that, I get:

```
pkg/gather/clusterconfig/clusterconfig.go:1094:26: invalid operation: p[month] (type *PodsWithAge does not support indexing)
```